features/partners.feature

```diff
@@ -247,35 +254,83 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
+      | voornamen | Karel  |
+      En de partner met burgerservicenummer '555550004' heeft de volgende aangaanHuwelijkPartnerschap gegevens
+      | naam  | waarde   |
+      | datum | 20091102 |
+      En de partner met burgerservicenummer '555550004' heeft GEEN 'ontbindingHuwelijkPartnerschap' gegevens
+      En de persoon heeft een partner met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550005 |
+      En de partner met burgerservicenummer '555550005' heeft de volgende naam gegevens
+      | naam      | waarde |
+      | voornamen | Björn  |
+      En de partner met burgerservicenummer '555550005' heeft de volgende aangaanHuwelijkPartnerschap gegevens
+      | naam  | waarde   |
+      | datum | 19870714 |
+      En de partner met burgerservicenummer '555550005' heeft de volgende ontbindingHuwelijkPartnerschap gegevens
+      | naam  | waarde   |
+      | datum | 20080706 |
       Als personen wordt gezocht met de volgende parameters
-      | naam                | waarde                          |
-      | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 999993380                       |
-      | fields              | partners                        |
-      Dan heeft de persoon met burgerservicenummer '999992806' alleen 'partners' met de volgende gegevens
-      | burgerservicenummer   | voornamen   |
-      | <burgerservicenummer> | <voornamen> |
+      | naam                | waarde                                               |
+      | type                | RaadpleegMetBurgerservicenummer                      |
+      | burgerservicenummer | 999991553                                            |
+      | fields              | partners.burgerservicenummer,partners.naam.voornamen |
+      Dan heeft de persoon met burgerservicenummer '999991553' alleen 'partners' met de volgende gegevens
+      | burgerservicenummer |
+      | 555550004           |
+      En heeft de partner met burgerservicenummer '555550004' de volgende 'naam' gegevens
+      | naam      | waarde |
+      | voornamen | Karel  |
 
-      Voorbeelden:
-      | omschrijving            | datum ontbinding | burgerservicenummer | voornamen |
-      | met bekende einddatum   | 20011109         | 555550003           | Osama     |
-      | met onbekende einddatum | 00000000         | 555550003           | Osama     |
 
   @proxy
-  Rule: Wanneer geen van de gegevens van de partner een waarde heeft, wordt de partner geleverd met type "PartnerOnbekend" en indicatieOnbekend met waarde true
-    - Dit is het geval wanneer elk gegeven van de partner leeg is of een standaardwaarde heeft
+  Rule: Wanneer geen van de gevraagde gegevens van de partner een waarde heeft, wordt er een 'partners' zonder gegevens geleverd.
+    - Dit is het geval wanneer elk gevraagd gegeven van de partner leeg is of een standaardwaarde heeft
     - Het gaat om de volgende standaardwaarden:
```

Comment:
dit moet eruit. zie https://github.com/VNG-Realisatie/Haal-Centraal-BRP-bevragen/pull/1050/files#r880224833

je vermengd nu twee regels:
1. bepaalde standaardwaarden moeten niet worden geleverd. Dit wordt uitgewerkt in onbekend_waardes.feature
2. wanneer na het verwijderen van standaardwaarden/onbekendwaarden er geen enkel gegeven van de partner nog met een waarde heeft, wordt de partner geleverd als leeg object. Dit is volgens mij waar deze rule over gaat.

Reply:
ook bevat de tabel onbekendwaarden voor datums (00000000). die worden wel geleverd in de vorm van een DatumOnbekend

Reply:
@fsamwel heeft aangegeven deze opmerking in PR #1129 mee te nemen